src/timer.rs

```diff
@@ -305,7 +309,7 @@ hal! {
     // feature = "stm32l4s9",
 ))]
 hal! {
-    TIM3:  (tim3, free_running_tim3, APB1R1, u16),
+    TIM3:  (tim3, free_running_tim3, tim3en, tim3rst, APB1R1, u32, timclk1),
```

Comment:
```suggestion
    TIM3:  (tim3, free_running_tim3, APB1R1, u16, timclk1),
```

Reply:
@reitermarkus Thank you, your suggestion is correct.